responsive-article/gulcin/index.html

```diff
@@ -0,0 +1,275 @@
+        <li class="nav-li selected"><a class="selected-a" href="#">Sign Up</a></li>
+        <li class="nav-li"><a href="#">Sign In</a></li>
+        <li class="nav-li account-symbol"><a href="#"><span class="material-symbols-outlined">
+              account_circle
+            </span></a></li>
+      </ul>
+      <div class="no-display">
+        <a href="#">Open in App</a><span class="material-symbols-outlined">
+          north_east
+        </span>
+      </div>
+    </nav>
+  </header>
+  <main>
+    <article class="main-article">
+      <h1>Time-saving CSS techniques to create responsive images</h1>
+      <div class="user-top">
+        <img class="user-img" src="images/iconn.jpg" alt="user picture">
+        <div class="user-text"><a href="#" class="underline">Andry Zaganel</a><a href="#" class="user-text-a">Follow</a></div>
+        <div class="user-text1">
+          <span>Published in &nbsp;<a href="#" class="underline" >We've moved to freeCodeCamp.org/news</a><span>&nbsp; &nbsp; &nbsp; 3 min read
+              &nbsp; &nbsp; &nbsp; &nbsp;Aug 12, 2018</span>
+        </div>
+      </div>
+      <div class="icons">
+        <div>
+          <a href="#"><i class="fa-solid fa-hands-clapping"></i></a><span>10.4K</span>
+          <a href="#"><i class="fa-regular fa-comment"></i></a><span>21</span>
+        </div>
+        <div>
+          <a href="#"><i class="fa-regular fa-bookmark"></i></a>
+          <a href="#"><i class="fa-regular fa-circle-play"></i></a>
+          <a href="#"><i class="fa-solid fa-upload"></i></a>
+        </div>
+      </div>
+      <figure class="large-img">
+        <picture>
+          <source srcset="images/picture1.medium.jpg" media="(max-width: 768px)" >
+          <source srcset="images/picture1.small.jpg" media="(max-width: 568px)">
+          <img src="images/picture1.jpg" alt="Photo by Kevin Ku on Unsplash" loading="lazy">
+        </picture>
+        <figcaption>Photo by Kevin Ku on Unsplash</figcaption>
+      </figure>
+
+      <p class="text-content">As a web developer, there is a high probability that you have encountered the two enemies of this article:
+        <strong>images</strong> and <strong> deadlines.</strong>Sometimes, for some reasons, your images won't fit the
+        layout and you don't want to wrap your head around this for hours.
+      </p>
+      <p  class="text-content">This situation has happened to me many times and I have learned from my mistakes. No more black magic hacks —
+        here are my five favorites techniques to handle image resizing.</p>
+      <h3  class="text-content">the “OMG I NEED THIS ASAP” way</h3>
+      <p class="text-content no-gap">It's 5:00 pm on Friday, you have to finish this page, but the images won't fit the layout. It's
+        time to use your magic trick!</p>
+      <picture>
+        <source srcset="images/code4.medium.jpg" media="(max-width: 768px)" >
```

Comment:
There's a <code> element in HTML, feel free to use it when you need to add a code snippet :)

Reply:
What's name of the element? I couldn't see :)